plugins/modules/source_control/gitlab/gitlab_runner.py

```diff
@@ -313,13 +333,23 @@ def main():
     access_level = module.params['access_level']
     maximum_timeout = module.params['maximum_timeout']
     registration_token = module.params['registration_token']
+    project = module.params['project']
 
     if not HAS_GITLAB_PACKAGE:
         module.fail_json(msg=missing_required_lib("python-gitlab"), exception=GITLAB_IMP_ERR)
 
     gitlab_instance = gitlabAuthentication(module)
+    if module._verbosity >= 2:
+        gitlab_instance.enable_debug()
```

Comment:
What's the effect of this?

Reply:
It provides actually useful debug output in case user requested it.

Reply:
But where does the debug output go? For using `module._verbosity` to make sense, it needs to be sent back to the controller.

Reply:
@drzraf ping ^